### PR TITLE
Removed PARANOID build option (#5631)

### LINKED ISSUE #5631: Remove PARANOID build option
Together with `#define ETH_PARANOIA`.

### PR DESCRIPTION
**Pull request by chrisjgf**
Fix #5631 - Remove PARANOID build and `#define ETH_PARANOIA`

### CODE REVIEW

File: libdevcore/OverlayDB.cpp [hunk 1]
```diff
@@ -140,7 +140,7 @@ bool OverlayDB::exists(h256 const& _h) const
 
 void OverlayDB::kill(h256 const& _h)
 {
-#if ETH_PARANOIA || 1
+#if 1
```
Comment (chfast) [inline libdevcore/OverlayDB.cpp]: The `#if 1` is useless now.

Comment (chrisjgf) [reply libdevcore/OverlayDB.cpp]: Sure thing, just removed the block

Comment (chfast) [reply libdevcore/OverlayDB.cpp]: Actually, you should not. Leave the `#if 1` block and remove the `#else` one.

Comment (chrisjgf) [reply libdevcore/OverlayDB.cpp]: Sorry, oversight on my behalf - just changed it.

**Comment by chfast** [comment]
Are you able to squash 2 last commits?

**Comment by chrisjgf** [comment]
> Are you able to squash 2 last commits?

Should now be squashed 👍

**Comment by chfast** [comment]
Ok, great. I will merge it as soon the Windows build finishes.

**Pull request merged**